Free Disk Space for NVHPC Actions

Problem:
```
No apport report written because the error message indicates a disk full error
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Errors were encountered while processing:
 /var/cache/apt/archives/nvhpc-25-7_25.7-0_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
Error: Process completed with exit code 100.
```

Solution:
```
      - name: Free Disk Space (Ubuntu)
        uses: jlumbroso/free-disk-space@main
        with:
          android: true
          dotnet: true
          haskell: true
          large-packages: true
          tool-cache: false
          swap-storage: true 
```

--- a/.github/workflows/nvhpc.yml
+++ b/.github/workflows/nvhpc.yml
@@ -25,6 +25,16 @@ jobs:
       - name: Get Sources
         uses: actions/checkout@v4.2.1
 
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@main
+        with:
+          android: true
+          dotnet: true
+          haskell: true
+          large-packages: true
+          tool-cache: false
+          swap-storage: true 
+
       - name: Install Dependencies
         shell: bash
         run: |

--- a/.github/workflows/nvhpc.yml
+++ b/.github/workflows/nvhpc.yml
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@v4.2.1
 
       - name: Free Disk Space (Ubuntu)
-        uses: jlumbroso/free-disk-space@main
+        uses: jlumbroso/free-disk-space@v1.3.0
         with:
           android: true
           dotnet: true

